themes/_default.yaml

```diff
@@ -321,13 +321,14 @@ dropdown:
     box-shadow: "0 0 5px 0 rgba(var(--tx-generic-color-shadow-rgb), .18)"
   select-arrow-zone:
     border-color-left: "var(--tx-generic-color-secondary-darker)"
-    color: "var(--tx-generic-color-active)"
+    border-style: "dotted"
```

Comment:
is this different per brand? if not, pls add this directly in the css without creating a variable.